index.d.ts

```diff
@@ -1,12 +1,13 @@
 import * as React from 'react';
+import FocusTrap from './lib/FocusTrap';
 
 type MouseTrapKeySequence = string | Array<string>;
 
 type KeyEventName = 'keyup' | 'keydown' | 'keypress';
 
 interface KeyMapOptions {
   sequence: MouseTrapKeySequence;
-  action: KeyEventName = 'keypress';
+  action: KeyEventName;
```

Comment:
I was under the impression what was there before was setting the default value to `keypress` (if not specified, the package behaves as if the `'keypress'` value was passed). Am I wrong in this?

Reply:
I don't believe so. Typescript is about definitions, not about action. Once transpiled, it's all just javascript and these interfaces won't even exist in your output. Maybe it'll be a future language feature?

You can set default values of function parameters (so long as the default is a literal of some sort), so maybe you're confusing it with that?

Reply:
Ah I see, yes I seem to have been mis-interpretting syntax that works for functional parameters and object literals as also being applicable for interfaces.